src/ui/region.coffee

```diff
@@ -18,15 +18,21 @@ initialize = (db) ->
 
   scaleg = actionbarRight.append('g')
       .attr('class', 'scale')
-      .attr('transform', "translate(65.5, 5.5)")
+      .attr('transform', "translate(82, 15)")
 
-  locationbuttong = actionbarRight.append('g')
+  locationbuttong = actionbarRight
+    .append('g')
+      .attr('transform', 'scale(2.6)')
+    .append('g')
       .attr('class', 'locationbutton')
-      .attr('transform', "translate(40, #{app.ACTIONBAR_HEIGHT / 2})")
+      .attr('transform', "translate(18, #{app.ACTIONBAR_HEIGHT / 2 / 2.6})")
 
-  noteG = actionbarRight.append('g')
+  noteG = actionbarRight
+    .append('g')
+      .attr('transform', 'scale(1.7)')
```

Comment:
Can we draw things at the right size instead of scaling them? It will be easier to maintain the code later if there are fewer transformations going on.